examples/segmentation/example_extract_clusters_normals.cpp

```diff
@@ -69,16 +69,16 @@ main (int, char **argv)
   std::cout << "Estimated the normals" << std::endl;
 
   // Creating the kdtree object for the search method of the extraction
-  pcl::KdTree<pcl::PointXYZ>::Ptr tree_ec  (new pcl::KdTreeFLANN<pcl::PointXYZ> ());
-  tree_ec->setInputCloud (cloud_ptr);
+  pcl::search::KdTree<pcl::PointXYZ>::Ptr tree;
```

Comment:
This shared pointer is being initialized to null.

Reply:
For pointers, I prefer auto: `auto name = make_shared<Type>(args);` (I always prefer auto :laughing: )